dataactvalidator/scripts/load_cfda.py

```diff
@@ -0,0 +1,16 @@
+from ftplib import FTP
+
+ftp = FTP('ftp.cfda.gov')      # connect to host, default port
+ftp.login()                    # user anonymous, passwd anonymous@
```

Comment:
We don't specify that username or password anywhere for the FTP connection correct? 

Also, comments on the same line can get messy. It's best to place the comments on the preceding lines.